src/python/pants/backend/python/subsystems/python_setup.py

```diff
@@ -67,6 +69,9 @@ def register_options(cls, register):
               'NOTE: this keyword is a temporary fix and will be reverted per: '
               'https://github.com/pantsbuild/pants/issues/5696. The long term '
               'solution is tracked by: https://github.com/pantsbuild/pex/issues/456.')
+    register('--allow-sdist-builds', advanced=True, type=bool, default=True, fingerprint=True,
+             help='Whether to allow building of distributions from source during 3rd party '
+               'resolution; Default: allow builds')
```

Comment:
The help string will render the default value, so no need to include it in this string.